Mathlib/LinearAlgebra/Matrix/Ideal.lean

```diff
@@ -0,0 +1,188 @@
+/-
```

Comment:
This file could also be `Mathlib.RingTheory.MatrixRing` or similar. If there are no desiderata I am unaware of, the present location seems fine.